# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: dom/indexedDB/PBackgroundIDBCursor.ipdl
# Commit: 8e9fb56cc22a
# Full Hash: 8e9fb56cc22a4a42925ce6dfe0abb6021a5b5afe
# Author: Birunthan Mohanathas <birunthan@mohanathas.com>
# Date: 2015-08-23 21:18:49
# Regressor Bug: 1168606
# File Overlap Count: 1
# Description:
#   Bug 1168606 - Part 4: Allow current key to be provided with PBackgroundIDBCursor::Continue. r=khuey
#   
#   The key is assumed to be unset for now.
# ==============================================================================

diff -r 1f31c24e2ca0 -r 8e9fb56cc22a dom/indexedDB/PBackgroundIDBCursor.ipdl
--- a/dom/indexedDB/PBackgroundIDBCursor.ipdl	Wed Aug 19 14:59:25 2015 -0700
+++ b/dom/indexedDB/PBackgroundIDBCursor.ipdl	Wed Aug 19 14:59:28 2015 -0700
@@ -77,7 +77,7 @@
 parent:
   DeleteMe();
 
-  Continue(CursorRequestParams params);
+  Continue(CursorRequestParams params, Key key);
 
 child:
   __delete__();
